packages/framework/src/Form/Admin/Complaint/ComplaintFormType.php

```diff
@@ -117,6 +135,38 @@ private function createBasicInformationGroup(
                 'multiple' => false,
                 'expanded' => false,
             ])
+            ->add('resolution', ChoiceType::class, [
+                'label' => t('Resolution'),
+                'required' => true,
+                'choices' => $this->complaintResolutionEnum->getAllIndexedByTranslations(),
```

Comment:
we should use `'choice_translation_domain' => false,` option here as the choices are already translated.

https://symfony.com/doc/current/reference/forms/types/choice.html#choice-translation-domain